tests/integration/test_models/test_full_battery_models/test_lithium_ion/test_spme.py

```diff
@@ -1,25 +1,16 @@
 #
 # Tests for the lithium-ion SPMe model
 #
-from tests import TestCase
 import pybamm
-import unittest
 from tests import BaseIntegrationTestLithiumIon
+import pytest
 
 
-class TestSPMe(BaseIntegrationTestLithiumIon, TestCase):
+class TestSPMe(BaseIntegrationTestLithiumIon):
+    @pytest.fixture(autouse=True)
     def setUp(self):
         self.model = pybamm.lithium_ion.SPMe
```

Comment:
Question: why do we need the fixtures here (and in other places)? Is this because `run_basic_processing_test` requires a model to be set up beforehand?

Reply:
Yes and everywhere else as well. Actually unittest identifies `setUp` as test setup function but for pytest we have to mark them explicitly so that they are available everywhere.

Reply:
I see – thanks for letting me know, resolving this conversation then!